libres/old_tests/enkf/test_enkf_update_log.cpp

```diff
@@ -12,10 +12,9 @@
 
 void test_obs_in_log() {
     int_vector_type *ens_active_list = int_vector_alloc(0, false);
-    bool_vector_type *ens_mask;
     int_vector_append(ens_active_list, 10);
 
-    ens_mask = int_vector_alloc_mask(ens_active_list);
+    const std::vector<bool> ens_mask(1, true);
```

Comment:
If we wanna be fancy C++17 developers
```suggestion
    const std::vector ens_mask{true};
```